lib/scripts/getDevDeps.js

```diff
@@ -0,0 +1,16 @@
+const execSync = require('child_process').execSync
+
+/**
+ * check for deployment environment variable ROOSEVELT_DEPLOYMENT
+ * if it exists don't install roosevelt devdeps
+ * otherwise, install devdeps
+ * @function installDevDeps
+ */
+;(function installDevDeps () {
+  if (process.env.ROOSEVELT_DEPLOYMENT) {
+    console.log('Skipping roosevelt dev dependencies.')
```

Comment:
Use the logger feature for these logs. [Example](https://github.com/rooseveltframework/roosevelt/blob/master/lib/scripts/configAuditor.js#L56)

Reply:
make sure to inject suitable emojis too